repository/zookeeper/docs/latest/configuration.md

```diff
@@ -0,0 +1,36 @@
+# Configuration 
+
+### Resources
+
+By default, KUDO Zookeeper resource configuration is set to the minimum recommended values for production usage. 
+But users can and should tune the configurations based on the workload requirements of their Kafka cluster.  
```

Comment:
```suggestion
But users can and should tune the configurations based on the workload requirements of whatever service makes use of Zookeeper. 
```

Reply:
Awesome. This is more generic information.